### PR TITLE
Fix spelling of parameterise and make consistent across book

### LINKED ISSUE #202: Consistent spelling of parameterise
Need to ensure spelling of parameterise is consistent across the book, and in line with the spelling used on gov.uk sites.

### PR DESCRIPTION
**Pull request by sarahcollyer**
Changed all spellings to parameterise/parameterised/parameterisation across the book to ensure consistency within the book, but also with other published works on gov.uk. The only exception is where we have referred to pytest.mark.parametrize, as this is a decorator and we can't change it!

Also had to fiddle with the line length for some of the chapters as the pre-commit hooks were failing. Have rendered the pages locally and they still look fine, but it's worth double checking I haven't borked something else in doing so.

### CODE REVIEW

File: book/checklist_higher.md [hunk 1]
```diff
@@ -99,7 +107,7 @@ Quality assurance checklist from [the quality assurance of code for analysis and
 - Stakeholder or user acceptance sign-offs are recorded near to the code.
 - Test are automatically run and recorded using continuous integration or git hooks.
 - The whole process is tested from start to finish using one or more realistic end-to-end tests.
-- Test code is clean an readable. Tests make use of fixtures and parametrisation to reduce repetition.
+- Test code is clean an readable. Tests make use of fixtures and parameterisation to reduce repetition.
```
Comment (ellie-o) [inline book/checklist_higher.md]: and misspelled

File: book/checklist_higher.md [hunk 2]
```diff
@@ -227,14 +248,15 @@ Quality assurance checklist from [the quality assurance of code for analysis and
...
+- [ ] Core functionality is unit tested as code. See [`pytest` for Python](https://docs.pytest.org/en/stable/) and
+ [`testthat` for R](https://testthat.r-lib.org/). 
 - [ ] Code based tests are run regularly.
 - [ ] Bug fixes include implementing new unit tests to ensure that the same bug does not reoccur.
 - [ ] Informal tests are recorded near to the code.
 - [ ] Stakeholder or user acceptance sign-offs are recorded near to the code.
 - [ ] Test are automatically run and recorded using continuous integration or git hooks.
 - [ ] The whole process is tested from start to finish using one or more realistic end-to-end tests.
-- [ ] Test code is clean an readable. Tests make use of fixtures and parametrisation to reduce repetition.
+- [ ] Test code is clean an readable. Tests make use of fixtures and parameterisation to reduce repetition.
```
Comment (ellie-o) [inline book/checklist_higher.md]: and misspelled

File: book/checklist_moderate.md [hunk 3]
```diff
@@ -92,7 +96,7 @@ Quality assurance checklist from [the quality assurance of code for analysis and
 - Stakeholder or user acceptance sign-offs are recorded near to the code.
 - Test are automatically run and recorded using continuous integration or git hooks.
 - The whole process is tested from start to finish using one or more realistic end-to-end tests.
-- Test code is clean an readable. Tests make use of fixtures and parametrisation to reduce repetition.
+- Test code is clean an readable. Tests make use of fixtures and parameterisation to reduce repetition.
```
Comment (ellie-o) [inline book/checklist_moderate.md]: and mispelled

File: book/checklist_moderate.md [hunk 4]
```diff
@@ -210,14 +225,15 @@ Quality assurance checklist from [the quality assurance of code for analysis and
...
+- [ ] Core functionality is unit tested as code. See [`pytest` for Python](https://docs.pytest.org/en/stable/) and
+ [`testthat` for R](https://testthat.r-lib.org/). 
 - [ ] Code based tests are run regularly.
 - [ ] Bug fixes include implementing new unit tests to ensure that the same bug does not reoccur.
 - [ ] Informal tests are recorded near to the code.
 - [ ] Stakeholder or user acceptance sign-offs are recorded near to the code.
 - [ ] Test are automatically run and recorded using continuous integration or git hooks.
 - [ ] The whole process is tested from start to finish using one or more realistic end-to-end tests.
-- [ ] Test code is clean an readable. Tests make use of fixtures and parametrisation to reduce repetition.
+- [ ] Test code is clean an readable. Tests make use of fixtures and parameterisation to reduce repetition.
```
Comment (ellie-o) [inline book/checklist_moderate.md:236]: and misspelled

**Review by ellie-o — CHANGES_REQUESTED** [review]
Change to parameterise is fine - noticed another spelling error

File: book/checklist_higher.md (hunk 2 above)
Comment (sarahcollyer) [reply book/checklist_higher.md]: Fixed!

File: book/checklist_moderate.md (hunk 3 above)
Comment (sarahcollyer) [reply book/checklist_moderate.md]: Fixed!

File: book/checklist_moderate.md (hunk 4 above)
Comment (sarahcollyer) [reply book/checklist_moderate.md:236]: Fixed!

**Comment by sarahcollyer** [comment]
Spotted typos all fixed; should all be good now @ellie-o .

**Review by ellie-o — APPROVED** [review]

**Pull request merged**